lib/rails_admin/config/model.rb

```diff
@@ -100,8 +100,8 @@ def pluralize(count)
 
       # Act as a proxy for the base section configuration that actually
       # store the configurations.
-      def method_missing(m, *args, &block)
-        send(:base).send(m, *args, &block)
```

Comment:
Naming/UncommunicativeMethodParamName: Method parameter must be at least 3 characters long.